flixel/system/render/gl/CanvasGL.hx

```diff
@@ -145,15 +145,28 @@ class CanvasGL extends GLDisplayObject
 		
 		var renderTransform:Matrix = __renderTransform;
 		
-	//	gl.enable(gl.SCISSOR_TEST);
-	//	gl.scissor(0, 0, _width, _height);
+		/*
```

Comment:
@gamedevsam this code is from my try to fix some issue with camera scissoring. i will remove it when i'll fix this issue